multisig/src/x25519.rs

```diff
@@ -43,6 +73,11 @@ impl PublicKey {
 }
 
 impl SecretKey {
+    pub fn public_key(&self) -> PublicKey {
+        let key = self.key.recover_public_key().expect("valid public key");
+        PublicKey { key }
+    }
+
```

Comment:
nit: implemented as `From<SecretKey> for PublicKey`? (doesn't really matter, just a personal preference i guess)